src/allotropy/parsers/utils/values.py

```diff
@@ -196,3 +197,20 @@ def num_to_chars(n: int) -> str:
 
 def str_or_none(value: Any) -> Optional[str]:
     return None if value is None else str(value)
+
+
+def get_val_from_xml(
+    xml_object: ElementTree.Element,
+    index_1: int,
```

Comment:
Is there not some way to retrieve these values by name instead of numerical index? 

Looking at the test data, they seem to be labelled. It seems like this would make it much easier to follow the code.

Reply:
Looks like `xml_object.find` will do this! You can even find values nested inside with a path-like value (e.g. `xml_object.find("name1/name2")`